src/test/java/SetTest.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 크기를 확인할 수 있어야 한다.")
+    void testGetSizeOfSet() {
+        assertEquals(3, numbers.size());
```

Comment:
assertj를 사용하면 검증 대상과 기댓값을 더 명시적으로 작성할 수 있고, 가독성이 좋다는 장점이 있습니다. :)
assertj에서 제공하는 컬렉션의 사이즈를 검증하는 메서드를 사용해보셔도 좋겠습니다.
```suggestion
        assertThat(numbers).hasSize(3);
```